index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="This is portfolio repository" />
+    <meta
+      name="author"
+      content="Bhaskar Ranjan Bora, B.Tech Computer Science"
+    />
+    <title>Bhaskar Ranjan Bora</title>
+
+    <link href="css/main.css" rel="stylesheet" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <main>
+      <header>
+        <div class="logo">
+          <h1>My Logo</h1>
+        </div>
+        <div class="button">
+          <i class="fa fa-bars fa-2x"></i>
+        </div>
```

Comment:
**[OPTIONAL]** -  Please wrap this in a `nav` tag. This would help improve the semantics of your site and also improve accessibility 😄 😄